packages/app/obojobo-repository/shared/components/notification.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../obojobo-document-engine/src/scss/_includes.scss';
+
+$poppin-pink: #af1b5c;
+$focus-pink: #fbdae6;
+$white: white;
+
+h1 {
+	margin: 0;
+}
+
+p {
+	display: block;
+}
```

Comment:
Try to nest these rules under whichever class rule below is/are relevant. There's too much of a chance that having them free-floating like this could cause conflicts elsewhere on the page.